app_customers/lib/pages/forfeit/forfeit_view_model.dart

```diff
@@ -0,0 +1,93 @@
+import 'dart:async';
+
+import 'package:rxdart/rxdart.dart';
+import 'package:trans_all_common_models/models.dart';
+
+import '../../data/repository/forfeitRepository.dart';
+
+/// The forfeit view model.
+class ForfeitViewModel {
+  final ForfeitRepository _forfeitRepository;
+
+  /// The list of forfeit.
+  List<Forfeit> listOfForfeit = [];
+
+  /// The list of operator type supported by forfeit.
+  Set<String> listOfOperatorType = {};
+
+  /// List of forfeit category.
+  Set<SelectedCategory> listOfForfeitCategory = {};
+
+  /// The filter list of forfeit.
+  BehaviorSubject<List<Forfeit>> filterListOfForfeit = BehaviorSubject();
+
+  /// List of forfeit validity.
+  Set<SelectedValidity> listOfForfeitValidity = {};
+
+  /// The variable for check if the initialization is done.
+  Completer<void> initializeDone = Completer<void>();
+
+  /// Constructs a new [ForfeitViewModel].
+  ForfeitViewModel(this._forfeitRepository) {
+    _initForfeit().then((_) => initializeDone.complete());
+  }
+
+  Future<void> _initForfeit() async {
+    final forfeits = await _forfeitRepository.getAllForfeit();
+    if (forfeits == null) return;
+    listOfForfeit.addAll(forfeits);
+    if (forfeits.isNotEmpty) {
+      final operator = forfeits.map((e) => e.reference.key).toSet();
+      listOfOperatorType.addAll(['⏚', ...operator]);
```

Comment:
we can replace '⏚' by all

Reply:
@warrenrhodes the issue with the text is an internationalisation